app/app.js

```diff
@@ -1,12 +1,22 @@
 const express = require('express')
 const app = express()
-const { getTopics } = require('./controllers/topics-controller')
+const { getTopics, getAllEndpoints } = require('./controllers/index')
 
 app.use(express.json())
 
 app.get('/api/topics', getTopics)
 
+app.get('/api', getAllEndpoints)
+
 //ERROR HANDLING
+app.use((err, req, res, next)=>{
+    //custom errors 
```

Comment:
This logic isn't needed yet. You could refactor to be our 500 catch all and log the error here